packages/ember-cli-eyeglass/README.md

```diff
@@ -45,6 +118,42 @@ const app = new EmberEngine(defaults, {
   }
 });
 ```
+ 
+Given the following folder structure:
+
+```
+my-engine/
+         /app/styles/
+                    /my-engine.scss
+                    /my-other-file.scss
+                    /_shared.scss
+         /addon/styles/
+                      /_shared.scss
+                      /my-engine.scss
+                      /secondary.scss             
+```
+
+If this is an eager engine:
+ 
+
+The contents of `my-engine/app/styles/my-engine.scss` will become `dist/assets/my-engine.css`
+The contents of `my-engine/app/styles/my-other-file.scss` will become `dist/assets/my-other-file.css`
+The contents of `my-engine/app/styles/_shared.scss` will only be included if `my-engine.scss` or `my-other-file.scss` explicitly import them.
+
+The contents of `my-engine/addon/styles/my-addon.scss` will be added to `dist/assets/vendor.css`
+The contents of `my-engine/addon/styles/secondary.scss` will be added to `dist/assets/vendor.css`
+The contents of `my-engine/addon/styles/_shared.scss` will only be included if `my-addon.scss` or `secondary.scss` explicitly import them.
+
+If this is a lazy engine:
+
+The contents of `my-engine/app/styles/my-engine.scss` will be ignored
```

Comment:
@chriseppstein I have no idea where these go, but they seem to be ignored. Future investigation can figure this out. But as we are just documenting todays behavior, I'll  likely leave the docs as is...

Reply:
agree.